specifications/credential.issuance.protocol.md

```diff
@@ -6,29 +6,27 @@ where a manual workflow is required.
 
 ## Issuance Flow
 
-The following sequence diagram depicts a non-normative flow where a client interacts with a [=Credential Issuer=] to
```

Comment:
We shouldn't create the impression that the diagram is normative. The steps below could be.